src/config.rs

```diff
@@ -0,0 +1,13 @@
+use serde::Deserialize;
+use std::path::PathBuf;
+
+#[derive(Deserialize, Debug)]
+pub struct Config {
+    pub update_interval: u64,
+    pub story_db: PathBuf,
+    pub webdriver_host: String,
+    pub webdriver_port: u16,
+    pub telegram_secret: String,
```

Comment:
Maybe move these fields into substructure?

Reply:
With serde(flatten) -- yeah, that's possible

Reply:
`envy` breaks on parsing u16 :(